### ISSUE TITLE
Strategy for serializing Annotations?

### ISSUE THREAD

**Issue by ChristianSchenk711** [emfjson/ecore.js#41]
I'm trying to use emfjson to serialize an ECore model which contains annotations to json and want to deserialize it using ecore.js. Unfortunately the key/value of the details are null after deserialization.

When checking, it looks like that:
EMFJson uses this format to serialize Annotations (copied from AnntationTest expectation):
```
        {"eClass":"http://www.eclipse.org/emf/2002/Ecore#//EClass","name":"Foo","eAnnotations": 
       [{"source":"source","details":{"displayName":"value"}}]}
```

ecore.js however serializes and deserializes like that:
```
details: [
{
eClass:"http://atest.com#//EStringToStringMapEntry"
key:"key1"
value:"value1"
}
]
```

Are there any options to customize that? What would be the suggested format? The EMFJson format?

**Comment by ghillairet** [comment]
This has been fixed in #42, the format for annotations is now the same as in emfjson-jackson. It is available in version 0.12.

**Comment by ChristianSchenk711** [comment]
Great, Thanks!
